src/Argu/Utils.fs

```diff
@@ -152,7 +172,7 @@ type UnionCaseInfo with
 
         match uci.GetCustomAttributes typeof<'T> with
         | [||] when includeDeclaringTypeAttrs ->
-            match uci.DeclaringType.GetCustomAttributes(typeof<'T>, false) with
+            match uci.DeclaringType.GetCustomAttributes(typeof<'T>, false) |> Seq.toArray with
```

Comment:
@eiriktsarpalis that's because .net core started as minimal **with a cleanup of api**. Instead of do like `Directoy.GetFiles` (array) vs `Directoy.EnumerateFiles` (ienumerable) in .net full, the fixed the api.

i think, ihmo, is better to evolve the api, if updating the code is easy, instead of adding duplicated functions.

But they got the issue about compatibility (drop in replacement), so each version of `netstandard` bring the api surface near the current one (`netstandard1.3` -> `1.5` -> `1.6`), so for `netstandard2.0` is pratically overlapping with `net45` and `mono`.